test/properties/second_quantization/electronic/resources/angular_momentum_op.json

```diff
@@ -1 +1,386 @@
-[["+_0 -_1 -_4 +_5", 1], ["+_0 -_2 -_4 +_6", 1], ["+_0 -_3 -_4 +_7", 1], ["-_0 +_1 +_4 -_5", 1], ["-_0 +_2 +_4 -_6", 1], ["-_0 +_3 +_4 -_7", 1], ["+_1 -_2 -_5 +_6", 1], ["+_1 -_3 -_5 +_7", 1], ["-_1 +_2 +_5 -_6", 1], ["-_1 +_3 +_5 -_7", 1], ["+_2 -_3 -_6 +_7", 1], ["-_2 +_3 +_6 -_7", 1], ["+_7 -_7", 0.75], ["+_6 -_6", 0.75], ["+_6 -_6 +_7 -_7", 0.5], ["+_5 -_5", 0.75], ["+_5 -_5 +_7 -_7", 0.5], ["+_5 -_5 +_6 -_6", 0.5], ["+_4 -_4", 0.75], ["+_4 -_4 +_7 -_7", 0.5], ["+_4 -_4 +_6 -_6", 0.5], ["+_4 -_4 +_5 -_5", 0.5], ["+_3 -_3", 0.75], ["+_3 -_3 +_7 -_7", -1.5], ["+_3 -_3 +_6 -_6", -0.5], ["+_3 -_3 +_5 -_5", -0.5], ["+_3 -_3 +_4 -_4", -0.5], ["+_2 -_2", 0.75], ["+_2 -_2 +_7 -_7", -0.5], ["+_2 -_2 +_6 -_6", -1.5], ["+_2 -_2 +_5 -_5", -0.5], ["+_2 -_2 +_4 -_4", -0.5], ["+_2 -_2 +_3 -_3", 0.5], ["+_1 -_1", 0.75], ["+_1 -_1 +_7 -_7", -0.5], ["+_1 -_1 +_6 -_6", -0.5], ["+_1 -_1 +_5 -_5", -1.5], ["+_1 -_1 +_4 -_4", -0.5], ["+_1 -_1 +_3 -_3", 0.5], ["+_1 -_1 +_2 -_2" , 0.5], ["+_0 -_0", 0.75], ["+_0 -_0 +_7 -_7", -0.5], ["+_0 -_0 +_6 -_6", -0.5], ["+_0 -_0 +_5 -_5", -0.5], ["+_0 -_0 +_4 -_4", -1.5], ["+_0 -_0 +_3 -_3", 0.5], ["+_0 -_0 +_2 -_2", 0.5], ["+_0 -_0 +_1 -_1", 0.5]]
```

Comment:
This change is unrelated. Please revert it

Reply:
I updated this to make the following test below pass: https://github.com/Qiskit/qiskit-nature/blob/748f96e4db4c86d90151398ef2791761b9bf5da7/test/properties/second_quantization/electronic/test_angular_momentum.py#L39-L53

Reply:
What exactly did you change then?

Reply:
It was formatted as a single line before. While this may not be perfect, the reformatting you did here makes it impossible to identify the changes to this file. Please make it a single line again so that the change can be identified.

We can reformat the json files to non-single lines in one go in a separate PR as this will likely be easier to handle in the future.

Reply:
Done.

Reply:
I think it's because of the change to `qiskit_nature/properties/second_quantization/electronic/angular_momentum.py` where `reduce` is replaced with `simplify`. So the operator is no longer normal-ordered.

Reply:
Oh thanks I had absolutely overlooked that change in this file!